pkg/verrazzano/edit/core.oam.dev.component/VerrazzanoWebLogicWorkload/ConfigurationDataTab/index.vue

```diff
@@ -1,16 +1,16 @@
 <script>
 // Added by Verrazzano
 import ConfigurationModelTab
-  from '@pkg/edit/core.oam.dev.component/VerrazzanoWebLogicWorkload/ConfigurationDataTab/ConfigurationModelTab';
+  from '@pkg/edit/core.oam.dev.component/VerrazzanoWebLogicWorkload//ConfigurationDataTab/ConfigurationModelTab';
 import IstioConfigTab
```

Comment:
Is the added slash intentional? What does the double-slash mean here?

Reply:
Fixed